contentcuration/contentcuration/frontend/channelEdit/components/edit/EditModal.vue

```diff
@@ -569,7 +575,7 @@
           })
         );
         this.creatingNodes = false;
-        this.$refs.inheritModal?.resetClosed();
```

Comment:
Now that we are not using the `resetClosed` public method, I think we can remove this from the `InheritAncestorMetadataModal` component